lib/html-proofer/log.rb

```diff
@@ -5,16 +5,23 @@ module HTMLProofer
   class Log
     include Yell::Loggable
 
-    def initialize(log_level)
-      @logger = Yell.new(format: false, \
-                         name: 'HTMLProofer', \
-                         level: "gte.#{log_level}") do |l|
-        l.adapter :stdout, level: %i[debug info warn]
-        l.adapter :stderr, level: %i[error fatal]
+
+    def initialize(log_level, hide)
+      if !hide
```

Comment:
I updated PR with this solution, but it also has its minuses - we have to be sure that we use only one instance of logger class, one that we created at Runner class and we have to check that @logger is not null every time we use it.